Deprecate apiserver-proxy and iam-kubeconfig (#13)
Demonsthere/kyma: docs/security/05-01-api-server-proxy.md

type: Configuration
---

> >**CAUTION:** The API Server PRoxy component (apiserver-proxy) is deprecated.

Reviewer: ```suggestion
>**CAUTION:** The API Server Proxy component (apiserver-proxy) is deprecated.
```

> >**CAUTION:** The API Server Proxy component (apiserver-proxy) is entering deprecation phase - a simpler solution is planned.

Reviewer 2: ```suggestion
>**CAUTION:** The API Server Proxy component (apiserver-proxy) is deprecated and will be replaced with a different mechanism.
```
